src/main/java/com/racingcar/game/GameMaker.java

```diff
@@ -1,17 +1,22 @@
 package com.racingcar.game;
 
 import com.racingcar.car.Car;
+import com.racingcar.view.model.GameInputDto;
+
+import java.util.ArrayList;
+import java.util.List;
 
 public class GameMaker {
-    public static Game make(int gameRound, int numberOfCar) {
-        return new Game(gameRound, makeCars(numberOfCar));
+    public static Game make(GameInputDto input) {
+        return new Game(input.getGameRound(), makeCars(input.getNames()));
     }
 
-    private static Car[] makeCars(int numberOfCar) {
-        Car[] cars = new Car[numberOfCar];
+    private static List<Car> makeCars(String names) {
+        List<Car> cars = new ArrayList<>();
```

Comment:
Stream을 애용하시는 것 같아서...ㅎㅎ 요런 방식도 가능할 것 같습니다
```suggestion
        return Arrays.stream(names.split(","))
                .map(Car::new)
                .collect(Collectors.toList());
```